### ISSUE TITLE
404 extra data found .. possible double count today

### ISSUE THREAD

**Issue by DesiCoder99** [COVID19Tracking/covid-tracking-data#32]
us_daily.csv reported cases & other data is double the expected amount. possible bug.
20200404	112	611510	2636104	31146	44316	76804	10414	1382	1312 ...

**Comment by johnrolfeellis** [comment]
For 20200404, states_daily_4pm_et.csv has four rows for each state. But only one of the four rows has non-zero values for deathIncrease, hospitalizedIncrease, negativeIncrease, positiveIncrease, totalTestResultsIncrease.

**Comment by Fumeng-Yang** [comment]
Yes, states_daily_4pm_et.csv has four rows for each state at this moment, and us_daily counts hospitalized etc. twice, which is absolutely scaring when I first saw the spike in the curve...

**Comment by johnrolfeellis** [comment]
states_daily_4pm_et.csv is now fixed.

**Comment by julia326** [comment]
Hi folks, thanks for your reporting here! Is it okay if I close this out now or still data issues in what you're seeing?
